enterprise/meta.go

```diff
@@ -45,6 +49,52 @@ func NewMetaClient(url *url.URL, InsecureSkipVerify bool, authorizer influx.Auth
 	}
 }
 
+type LDAPConfig struct {
+	Enabled bool `toml:"enabled"`
+}
+
+func (m *MetaClient) RequestLDAPChannel(ctx context.Context, errors chan error) chan *http.Response {
+	channel := make(chan *http.Response, 1)
+	go (func() {
+		res, err := m.Do(ctx, "/ldap/v1/config", "GET", m.authorizer, nil, nil)
```

Comment:
What do you think about returning the bytes from ioutil.ReadAll(res.Body) rather than the response?

Also, you'll need to do a defer close on the body,

Reply:
good idea